CMakeLists.txt

```diff
@@ -292,6 +288,14 @@ message(STATUS "NGRAPH_INTERPRETER_STATIC_LIB_ENABLE: ${NGRAPH_INTERPRETER_STATI
 message(STATUS "NGRAPH_CPU_STATIC_LIB_ENABLE:         ${NGRAPH_CPU_STATIC_LIB_ENABLE}")
 message(STATUS "NGRAPH_DYNAMIC_COMPONENTS_ENABLE:     ${NGRAPH_DYNAMIC_COMPONENTS_ENABLE}")
 
+if("${CMAKE_GENERATOR}" STREQUAL "Ninja" AND (NGRAPH_TOOLS_ENABLE OR NGRAPH_CPU_ENABLE OR NGRAPH_MLIR_ENABLE OR
+        NGRAPH_INTELGPU_ENABLE OR NGRAPH_GPU_ENABLE OR NGRAPH_INTERPRETER_ENABLE OR NGRAPH_NOP_ENABLE OR
+        NGRAPH_GPUH_ENABLE OR NGRAPH_GENERIC_CPU_ENABLE OR NGRAPH_ONNX_IMPORT_ENABLE OR NGRAPH_DEX_ONLY OR
+        NGRAPH_ENABLE_CPU_CONV_AUTO OR NGRAPH_CODE_COVERAGE_ENABLE OR NGRAPH_PLAIDML_ENABLE OR NGRAPH_JSON_ENABLE))
+    message(FATAL_ERROR "Ninja generator not supported.")
```

Comment:
Not sure if you need "NGRAPH_ENABLE_CPU_CONV_AUTO", "NGRAPH_CODE_COVERAGE_ENABLE" , "NGRAPH_DEX_ONLY"

Reply:
Ok, I will remove it. Just added it because I wasn't sure that Ninja is supported for these options.

Reply:
Fixed